documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -613,6 +613,10 @@ The following configuration properties are _required_ unless a default value is
 |
 |Password to be used when connecting to MongoDB. This is required only when MongoDB is configured to use authentication.
 
+|`mongodb.authSource`
```

Comment:
Changed this to `mongodb.authsource` to be consistent with what's in the code and what also is our general convention.